Sources/TuistKit/Generator/Generator.swift

```diff
@@ -264,6 +264,9 @@ class Generator: Generating {
         )
         let models = (workspace: workspace, projects: projects)
 
+        // Check circular dependencies
+        try GraphLoaderLinter().lintProject(at: path, projects: projects)
```

Comment:
I'd recommend injecting the linter through the constractor of the generator so that we can easily unit test that these calls are happening.

Reply:
Oops, yes sorry. I think I was in the middle of it and forgot 🥲